go.mod

```diff
@@ -634,3 +648,6 @@ exclude (
 )
 
 replace github.com/github/smimesign => github.com/grafana/smimesign v0.2.1-0.20220408144937-2a5adf3481d3
+
+// To be used until upstream PR of fixes is merged https://github.com/webdevops/azure-metrics-exporter/pull/41
+replace github.com/webdevops/azure-metrics-exporter => github.com/kgeckhart/azure-metrics-exporter v0.0.0-20230124174512-16e25d5c8d3b
```

Comment:
nit: move to section starting at 601 where all the exporters replaces are defined?